projects/storefrontlib/src/lib/product/components/product-tabs/container/product-tabs.component.html

```diff
@@ -0,0 +1,42 @@
+<div class="details" *ngIf="(product$ | async) as product">
+  <h3 #descriptionHeader (click)="select($event, description)">
```

Comment:
We need to keep the inner outlets for details, specs, review and shipping for as long as we haven't put #1484 into place. Otherwise customers cannot customise these bits easily.